services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -2756,15 +2734,7 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                required:
-                  - data
-                properties:
-                  data:
-                    $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/put/requestBody/content/application~1json/schema'
-                  error:
-                    nullable: true
-                    default: null
+                $ref: '#/paths/~1projects~1%7Bproject_id%7D~1nodes~1%7Bnode_id%7D~1resources/get/responses/200/content/application~1json/schema'
```

Comment:
Changed from `put` to `get` here as well.

Reply:
this one is weird

Reply:
but as is the one above

Reply:
but I think it is ok, the object is correct. both get and put have the same one.